Author: Steve Kemp <skx@debian.org>
Description: Missing openssl  includes in sshcrypto.c. 
This patch was through diff.gz and now is implemented as a dpatch.
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/

--- a/sshcrypto.c
+++ b/sshcrypto.c
@@ -14,6 +14,8 @@
 
 #include <sys/types.h>
 #include <openssl/ssl.h>
+#include <openssl/blowfish.h>
+#include <openssl/des.h>
 
 #include <err.h>
 #include <stdio.h>
